hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1128,3 +1129,55 @@ def hpat_pandas_series_le_impl(self, other):
         return hpat_pandas_series_le_impl
 
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
+
+
+@overload_method(SeriesType, 'max')
+def hpat_pandas_series_max(self, axis=0, skipna=True, level=None, numeric_only=None):
```

Comment:
Parameters are diffrent https://pandas.pydata.org/pandas-docs/version/0.25/reference/api/pandas.Series.max.html?highlight=max#pandas.Series.max
Please fix the parameters

Reply:
parameter `skipna=True` is still diffrent

Reply:
Sorry, looked into pandas sources and found
```
def max(self, axis=None, skipna=True, *args, **kwargs):
```
it looks like they have documentations issue